PIM/Integration/02_ManageAttributeSets.md

```diff
@@ -0,0 +1,272 @@
+
+  ![Attribute set created](../../Assets/Screenshots/PIM/Settings/AttributeSets/AttributeSetCreated.png "[Attribute set created]")  
+
+5. Click the ![Back](../../Assets/Icons/Back02.png "[Back]") (Back) button in the upper left corner to return to the list of attribute sets or proceed to [edit the attribute set](#edit-an-attribute-set).
+
+
+
+## Edit an attribute set
+
+After you have created an attribute set, you can edit it. Depending on the selected inheritance type, only a certain number of attributes may be editable. However, you can add further attributes to an attribute set or remove unmapped attributes from an attribute set.
+
+### Add an attribute to the set
+
+Add a new attribute to an attribute set to define any further specifications.
+
+#### Prerequisites
+
+- An attribute set has been created, see [Create an attribute set](#create-an-attribute-set).
+- At least one additional attribute has been created, see [Create an attribute](./01_ManageAttributes.md#create-an-attribute).
+
+> [Info] By default, the *PIM Basic Set* attribute set has been predefined in the *Core1 Platform*.
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTE SETS*
+
+![Attributes](../../Assets/Screenshots/PIM/Settings/AttributeSets/AttributeSets.png "[Attributes]")
+
+1. Click the attribute set you want to edit in the list of attribute sets.   
+  The *Edit Attribute Set* view is displayed.
+
+  ![Edit attribute set](../../Assets/Screenshots/PIM/Settings/AttributeSets/EditAttributeSet.png "[Edit attribute set]")
+
+2. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the *Assigned Attributes* section.   
+  The *Add attributes* view is displayed. All active attributes that are not yet included in the selected attribute set are displayed in the list.
+
+  ![Add attribute](../../Assets/Screenshots/PIM/Settings/AttributeSets/AddAttributes.png "[Edit attribute set]")
+
+  > [Info] If the **There are no unassigned attributes.** notice is displayed in the *Add attributes* view, all active attributes are already included in the selected attribute set. Check, if you have selected the appropriate attribute set, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list or [create a new attribute](./01_ManageAttributes.md#create-an-attribute).
+
+3. Select the checkboxes of the attributes in the list, you want to add to the attribute set.   
+  The editing toolbar is displayed above the list.
+
+4. Click the [ADD] button in the editing toolbar.   
+  The selected attributes have been added to the attribute set. The *Add attributes* view is closed. The *Edit Attribute Set* view is displayed again. The added attributes are displayed in the *Assigned Attributes* section.
+
+5. Click the [SAVE] button.   
+  The changes have been saved. The *Edit Attribute Set* view is closed.
```

Comment:
It is not necessary to click the save button. The attributes are already added to the AttributeSet in the 4th step.